ci/integration/logging/run_validation.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/env bash
+# CI script for UBI8 job
+# purpose: verify the expected logging data is saved in supported storage types
+
+
+set -ex
+
+dnf install -y jq hostname
+
+TS=$(date +'%Y.%m.%d')
+HOST=$(hostname)
+
+ELASTIC_URL=http://127.0.0.1:9200
+LOKI_URL=http://127.0.0.1:3100
+
+######################### validate elasticsearch data #########################
+# debug output of cluster status
+curl -sX GET "$ELASTIC_URL/_cluster/health?pretty"
+# verify expected index
+#TODO(mmagr): adapt elasticseatch plugin to create index templates avoinding unnecessary prefix and suffix
+expected_index="sglogs-$(echo $HOST | tr - _).$TS"
+curl -sX GET "$ELASTIC_URL/_cat/indices/sglogs-*?h=index"
+found_index=$(curl -sX GET "$ELASTIC_URL/_cat/indices/sglogs-*?h=index")
+[[ "${found_index}" =~ "${expected_index}" ]] || exit 1
+# debug output of index content
+curl -sX GET "$ELASTIC_URL/${expected_index}/_search?pretty" -H 'Content-Type: application/json' -d'
+{
+  "query": {
+    "match_all": {}
+  }
+}
+'
+# verify expected documents
+res=$(curl -sX GET "$ELASTIC_URL/${expected_index}/_search" -H 'Content-Type: application/json' -d"
```

Comment:
Would it just make more sense to use single-quotes for the query so you don't have to escape the double-quotes? Maybe there is something going on here I don't understand though.

Reply:
That is a left over when I was trying to query log records not only by phrase, but also by host (saved in env variable) and by beginning of timestamp (also saved in env var). But I failed miserably on that front. Since I had al the queries written with double quotes double quotes. But yeah I'll fix it.